garak/probes/continuation.py

```diff
@@ -73,6 +76,17 @@ def __init__(self, config_root=_config):
                     self.prompts.append(candidate_prompt.strip())
                     self.triggers.append(term)
 
+        if self.follow_prompt_cap:
+            num_ids_to_delete = max(
+                0, len(self.prompts) - config_root.run.soft_probe_prompt_cap
+            )
```

Comment:
Cannot assume `config_root` is the global module `_config`.

Reply:
this has been an impactful 9-token reply